components/teams-service/storage/conformance/conformance_test.go

```diff
@@ -405,20 +378,21 @@ func testRemoveUsers(ctx context.Context, t *testing.T, s storage.Storage) {
 	require.NoError(t, err, "failed to read back teams")
 	assert.Equal(t, 1+len(storage.NonDeletableTeams), len(teams))
 
-	result, err := s.GetTeamByName(ctx, name)
+	_, err = s.GetTeam(ctx, id)
 	require.NoError(t, err, "failed to read back team")
-	userIDs, err = s.GetUserIDsForTeam(ctx, result.ID)
+
+	userIDs, err = s.GetUserIDsForTeam(ctx, id)
 	require.NoError(t, err)
 	assert.ElementsMatch(t, []string{one, three}, userIDs)
 }
 
 func testRemoveUsersNotFound(ctx context.Context, t *testing.T, s storage.Storage) {
-	_, err := s.RemoveUsers(ctx, uuid.Must(uuid.NewV4()), []string{"user-1"})
+	_, err := s.RemoveUsers(ctx, "not-found", []string{"user-1"})
 	assert.Equal(t, storage.ErrNotFound, err)
 }
 
 func testGetTeam(ctx context.Context, t *testing.T, s storage.Storage) {
-	team, err := s.StoreTeam(ctx, "name", "description")
+	team, err := s.StoreTeam(ctx, "id", "name", []string{})
```

Comment:
Consider adding a named constant to make all these tests a wee bit clearer:
```suggestion
	team, err := s.StoreTeam(ctx, "id", "name", emptyProjectsList)
```